package.json

```diff
@@ -59,13 +59,13 @@
     "ajv": "^4.11.4",
     "avvio": "^0.3.0",
     "body": "^5.1.0",
-    "fast-json-stringify": "^0.10.3",
-    "fast-safe-stringify": "^1.1.6",
+    "fast-json-stringify": "^0.10.4",
+    "fast-safe-stringify": "^1.1.11",
     "fastseries": "^1.7.2",
     "middie": "^0.1.1",
     "pathname-match": "^1.2.0",
     "pino-http": "^2.4.2",
     "pump": "^1.0.2",
-    "wayfarer": "^6.4.1"
+    "wayfarer": "^6.5.0"
```

Comment:
ncu is ok.  Are you using it with --upgradeAll? In theory all those changes should not be happening: https://www.npmjs.com/package/npm-check-updates#why-is-it-not-updating-100-to-101-when-101-is-the-latest.

I'm ok with greenkeeper.